### PR TITLE
refactored CSS files in src/screens/OrganizationVenues

### LINKED ISSUE #2519: Refactor CSS files in src/screens/OrganizationVenues
# Refactor CSS files in src/screens/OrganizationVenues

To improve the UI/UX of Talawa-Admin we have been working on:
1. making the application more suitable to color blind users
2. streamlining all CSS into a single global file

## Additional information:
1. The foundational work for this was completed in this PR:
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/2466
2. The desired single CSS file is:
    - `src/style/app.module.css`

## The challenge:
The goal is to convert the CSS file in this subdirectory and all the components related to this screen to use this new design pattern. This means that:
1. All CSS files in this subdirectory and all the CSS files in related components must be merged into the global CSS file using pre-existing CSS.
2. All CSS files in this subdirectory and all the CSS files in related components must be deleted after the merge.
3. The remaining relevant files in this subdirectory must only reference the global CSS file.
4. All the CSS files in the related components of this subdirectory must adhere to the same guidelines.

### PR DESCRIPTION
**Pull request by antriksh-9**
**What kind of change does this PR introduce?**
refactored CSS files in src/screens/OrganizationVenues

**Issue Number:**

Fixes #2519 

**Did you add tests for your changes?**
No

**Snapshots/Videos:**

Screenshot 2024-12-26 031813

**If relevant, did you update the documentation?**
Not relevant

**Summary**
The goal is to convert the CSS file in this subdirectory and all the components related to this screen to use this new design pattern.

**Does this PR introduce a breaking change?**
No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

## Summary by CodeRabbit

- **Bug Fixes**
	- Removed outdated styles for the organization venues interface, which may affect visual layout.

- **New Features**
	- Updated color variables and hover/focus styles to enhance UI components' visual consistency and interactivity.
	- Improved interactivity and visual feedback for dropdown and button components.
	- Introduced new styles for the capacity label and enhanced hover effects for cards.
	- Adjusted styling sources for the VenueCard component to improve visual presentation.

### CODE REVIEW

File: src/style/app.module.css [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+  --grey-bg-color: #b0b0b0;
   --subtle-blue-grey: #7c9beb;
   --subtle-blue-grey-hover: #5f7e91;
   --modal-width: 670px;
   --modal-max-width: 680px;
   --input-shadow-color: #dddddd;
   --delete-button-bg: #f8d6dc;
   --delete-button-color: #ff4d4f;
-  --search-button-bg: #a8c7fa;
+  --search-button-bg: #0071e3;
```
Comment (palisadoes) [inline src/style/app.module.css:12]: Why was this color changed when we are going towards a non green color scheme?

File: src/style/app.module.css [hunk 2]
```diff
@@ -193,24 +193,33 @@
...
+
 .dropdownItem:hover,
 .dropdownItem:focus,
 .dropdownItem:active {
-  background-color: var(--dropdown-hover-color) !important;
+  background-color: var(--dropdown-hover-color, #e0e0e0) !important;
   color: var(--brown-color) !important;
   outline: none !important;
-  box-shadow: none !important;
+  box-shadow: 0 0 4px var(--highlight-color, #0071e3);
```
Comment (palisadoes) [inline src/style/app.module.css:222]: Same here

File: src/style/app.module.css [hunk 3]
```diff
@@ -193,24 +193,33 @@
...
+
 .dropdownItem {
   background-color: white !important;
   color: var(--brown-color) !important;
   border: none !important;
 }
 
+.dropdownItem:focus,
+.dropdownItem:hover {
+  outline: 2px solid var(--highlight-color, #0071e3);
```
Comment (palisadoes) [inline src/style/app.module.css:213]: Same here. The color doesn't match the current design

File: src/style/app.module.css [hunk 4]
```diff
@@ -193,24 +193,33 @@
...
 
 .dropdown:is(:hover, :focus, :active, :focus-visible, .show) {
-  background-color: transparent !important;
+  background-color: var(--dropdown-hover-color, #e0e0e0);
   border: 1px solid var(--brown-color);
   color: var(--brown-color) !important;
 }
 
+.dropdown:is(:focus, :focus-visible) {
+  outline: 2px solid var(--highlight-color, #0071e3);
```
Comment (palisadoes) [inline src/style/app.module.css:202]: Same here. The color doesn't match the current design

**Review by palisadoes — CHANGES_REQUESTED** [review]
We spent significant time converting from the predominantly green color scheme to one that is color blind friendly. Please adjust your colors to those that already exist. We are using blue / grey tones in most cases.

File: src/style/app.module.css [hunk 5]
```diff
@@ -1,15 +1,15 @@
 :root {
-  --brown-color: #555555;
+  --brown-color: #333333;
```
Comment (palisadoes) [inline src/style/app.module.css:2]: Do not change this color. This will impact the existing design

File: src/style/app.module.css [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
   --dropdown-hover-color: #eff1f7;
-  --grey-bg-color: #eaebef;
+  --grey-bg-color: #b0b0b0;
```
Comment (palisadoes) [inline src/style/app.module.css:4]: Do not change this color. This will impact the existing design

File: src/style/app.module.css [hunk 7]
```diff
@@ -193,24 +193,33 @@
 }
 
 .dropdown:is(:hover, :focus, :active, :focus-visible, .show) {
-  background-color: transparent !important;
+  background-color: var(--dropdown-hover-color, #e0e0e0);
```
Comment (palisadoes) [inline src/style/app.module.css:196]: Why was this changed?

File: src/style/app.module.css [hunk 8]
```diff
@@ -339,15 +348,23 @@
   align-items: center;
 }
 
+.searchButton:hover {
+  background-color: var(--search-button-hover-bg, #005bb5);
```
Comment (palisadoes) [inline src/style/app.module.css:352]: This color seems too dark for the current color scheme. Use a pre-existing color value

File: src/style/app.module.css [hunk 9]
```diff
@@ -339,15 +348,23 @@
...
 .addButton {
   margin-bottom: 10px;
   background-color: var(--search-button-bg);
   border-color: var(--grey-bg-color);
-  color: #555555;
 }
 
 .addButton:hover {
-  background-color: #286fe0;
+  background-color: #005bb5;
```
Comment (palisadoes) [inline src/style/app.module.css:367]: Use the previous color

File: src/style/app.module.css [hunk 10]
```diff
@@ -339,15 +348,23 @@
   align-items: center;
 }
 
+.searchButton:hover {
+  background-color: var(--search-button-hover-bg, #005bb5);
+  border-color: var(--search-button-hover-border, #333333);
```
Comment (palisadoes) [inline src/style/app.module.css:353]: Use a pre-existing border color

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comments

**Comment by palisadoes** [comment]
Please fix the failing tests

**Comment by antriksh-9** [comment]
@palisadoes 
fixed failed tests.

File: src/style/app.module.css [hunk 11]
```diff
@@ -61,6 +50,43 @@
...
+  grid-column: auto / span 2;
+}
+
+.cards:first-child:nth-last-child(even),
+.cards:first-child:nth-last-child(even) ~ .box {
+  grid-column: auto / span 1;
+}
+
+.capacityLabel {
+  background-color: #31bb6b !important;
```
Comment (palisadoes) [inline src/style/app.module.css:75]: 1. We are migrating away from the green theme as part of our support for color blind users.
2. Take cues from other similar elements in this file and the PR mentioned in the issue.
    1. https://github.com/PalisadoesFoundation/talawa-admin/pull/2466

**Review by palisadoes — CHANGES_REQUESTED** [review]
see comment

**Comment by palisadoes** [comment]
Please make CodeRabbit.ai approve your changes

**Review by palisadoes — APPROVED** [review]

**Pull request merged**